src/c/llvmgcov.cpp

```diff
@@ -99,7 +98,27 @@ void parse_llvm_gcno(char* working_dir, char* file_stem, uint8_t branch_enabled)
   );
 
   CustomGCOVFile GF;
+  std::string GCNO = std::string(file_stem) + ".gcno";
+
+  GCOVBuffer GCNO_GB(GCNO_Buff);
+  if (!GF.readGCNO(GCNO_GB)) {
+    errs() << "Invalid .gcno File!\n";
+    return;
+  }
+
+  GCOVBuffer GCDA_GB(GCDA_Buff);
```

Comment:
There are cases where there's only a gcno file and no gcda file (e.g. if a file is never executed). We should make sure this case is still supported.